src/dash-adapter.js

```diff
@@ -1057,8 +1058,9 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
   _getParsedImageTracks(): Array<ImageTrack> {
     if (this._manifestParser) {
       const imageSet = this._manifestParser.getImageSet();
+      const middleBaseURL = this._manifestParser.getBaseUrl();
```

Comment:
middleBaseUrl is a string that's being inserted into the middle of a url ? what does it represent ? would it usually be empty when we don't use redirect ?